app/views/content/home.md

```diff
@@ -5,10 +5,10 @@
     Read official Department for Education information to help you become a teacher. Learn about funding, teacher training, and steps to become a teacher. 
   date: "2021-06-16"
   fullwidth: true
-  subtitle: "Get information and support to help you become a teacher."
-  subtitle_button: "Register for updates"
-  subtitle_link: "/mailinglist/signup/name"
-  image: "media/images/content/hero-images/0012.jpg"
+  subtitle: "Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event."
+  subtitle_button: "Get Into Teaching events"
+  subtitle_link: "/events?teaching_events_search[type][]=git"
```

Comment:
@ethax-ross Can you change this link to https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events please - we want people to go to the showcase page not a filtered list on events listing page